.golangci.yml

```diff
@@ -122,6 +122,7 @@ linters:
     - nilnil
     - paralleltest # [too many false positives] detects missing usage of t.Parallel() method in your Go test
     - tagliatelle
+    - testpackage # Disable testpackage linter as we want to use e2e_test package name
```

Comment:
Can we keep that linter and only switch it off for e2e_test ? 

Like with

```yaml
linters-settings:
  testpackage:
    skip-files:
      - "e2e/*.go"
```

or in code with

```golang
//nolint:testpackage
package e2e_test
```